### PR TITLE
[JENKINS-37838] Calls to backend MUST end with /

### PR DESCRIPTION
**Pull request by imeredith**
# Description

This fixes current master with safari. Needs testing with IE still.

See [JENKINS-37838](https://issues.jenkins-ci.org/browse/JENKINS-XXXXX).

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Apppropriate unit or acceptance tests or explaination to why this change has no tests
- [x] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [x] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explaination given

@jenkinsci/code-reviewers @reviewbybees

### CODE REVIEW

**Comment by kzantow** [comment]
Safari isn't following redirects? Pretty sure I tested it. Oh well, good change & :bee:

**Comment by michaelneale** [comment]
@kzantow It did before, what I think is that the JWT token was tripped up by it missing the "/". This also reduces a round trip so ACE!!!!

File: blueocean-dashboard/src/main/js/util/UrlUtils.js [hunk 1]
```diff
@@ -174,14 +174,14 @@ export function getRestUrl({ organization, pipeline, branch, runId }) {
     let url = `${jenkinsUrl}/blue/rest/organizations/${encodeURIComponent(organizationName)}`;
     if (pipelineName) {
         // pipelineName might include a folder path, don't encode it
-        url += `/pipelines/${pipelineName}`;
+        url += `/pipelines/${pipelineName}/`;
```
Comment (kzantow) [inline blueocean-dashboard/src/main/js/util/UrlUtils.js]: 🐛  probably only need to append a `/` at the very end, line 186

**Comment by kzantow** [comment]
A shame Safari has an issue following redirects! Grr, thought I tested it.

**Comment by imeredith** [comment]
@kzantow better?

**Comment by kzantow** [comment]
@imeredith excellent :bee: :)

**Pull request merged**